clinicaltrials/frontend/management/commands/load_data.py

```diff
@@ -526,20 +531,22 @@ def convert_to_csv():
             td["pending_results"] = does_it_exist(soup.pending_results)
 
             td["pending_data"] = dict_or_none(parsed_json, [cs, "pending_results"])
+                                  
+            date_cond1 = td["available_completion_date"]
+                                  + relativedelta(years=1)
```

Comment:
Wondering if I screwed up the indentation here and that's what is making the tests fail.

Reply:
Nope, something else.